editor/components/block-list/style.scss

```diff
@@ -1,46 +1,47 @@
+.editor-block-list__layout {
+	// prevent collapsing margins on container
+	padding-top: .1px;
```

Comment:
I'm not seeing any effective changes of removing this style. What is this collapsing into?

Also noting that `.editor-block-list__layout` is wrapping a nested container, so these styles should apply there as well.

Reply:
Back when the initial purpose of this PR was to allow margins to collapse, I could swear there was an issue with the columns block and adjecent blocks that was fixed by adding this rule.

However since that's been pulled out, for now I'm going to also remove this rule. Since I hope to revisit collapsing margins in a future PR, I can always restore the rule if I recall what it did.